src/cms/src/api/emergency-alert/content-types/emergency-alert/lifecycles.js

```diff
@@ -1,23 +1,27 @@
 "use strict";
 
+const formatDateToPacificTime = (dateString) => {
+  const date = new Date(dateString)
+  const localDate = date.toLocaleDateString('en-CA', { timeZone: 'America/Los_Angeles' })
+  return new Date(localDate).toISOString().split('T')[0]
```

Comment:
this `toLocaleDateString` function already returns a YYYY-MM-DD date string; can't we just return that without needing this last line that parses it and formats it again?

Reply:
This `localDate` returns MM/DD/YYY, like `12/11/2024` so it still needs to be converted 😓

Reply:
In my browser, `en-CA` returns YYYY-MM-DD but `en-US` returns MM/DD/YYYY 🤔 Apparently that can change(??) Maybe because I've got my machine set to use YYYY-MM-DD date formats in the OS? Since this project is already using `date-fns`, you could use that here to get a predictable output (see parkAccessStatus.js where date-fns `format` function is used to output a date with year-month-day format)

Reply:
also, what format is the `dateString` parameter when it gets passed in? Let's add some comments to explain what this function does

Reply:
Thanks! Using `date-fns` seems to be a good idea. Strapi returns an ISO date string like `2025-01-01T00:00:00.000Z`, but it's interesting I needed to convert it into YYYY-MM-DD format when updating data in the Strapi lifecycle.